### ISSUE TITLE
Transaction detail should show current transaction info

### ISSUE THREAD

**Issue by lsrom** [ghostfolio/ghostfolio#528]
Opening transaction detail shows average buying price, all time performance of all transactions of this stock and total investment. While this is useful and should be accessible somehow, transaction detail should show price of the stock at the time of transaction (not average of all) and performance for this transaction only. Especially the last part is important since you might want to know single transaction performance when you are taxing sold stocks.

TLDR: data for all transaction of given stock and data of a single transaction should be two separate screens.

**Comment by dtslvr** [comment]
Hi @lsrom

Thank you very much for posting your idea.

In [Ghostfolio](https://ghostfol.io), we are currently only using a position detail dialog, independent of where this has been opened (e.g. from position overview or transactions table). To keep things simple, what do you think of extending this position detail dialog by a transactions table (entries filtered by symbol of course, displaying quantity, unitPrice and in a next step even the performance)?

Are you interested in contributing to this project?

**Comment by lsrom** [comment]
I think current dialog is fine for position overview as it shows what you expect. Clicking one transaction should however show data ONLY about this transaction (change of value for this particular buy, performance, buy price, current price and quantity, ....).

It should be as simple as not opening PositionDialog from `transactions-table.component.ts` or at least feeding it different data. 

I haven't touched Angular in a long time but I do have some experience with it so I could contribute but I am still not sure if I want to commit to this project, sorry. Currently I am using google sheet to track my portfolio and while this definitely looks better it can't beat the flexibility of google sheet. Sorry for just dumping bunch of requests on you, I'll lurk around some more and maybe send some MR your way someday. No promises though.

**Comment by dtslvr** [comment]
The position detail dialog has been extended by a transactions table:

<img width="1224" alt="position-details-with-transactions" src="https://user-images.githubusercontent.com/4159106/147574713-b9377046-f6ae-443e-a90e-60d82b936dda.png">
